SCYLLA-VERSION-GEN

```diff
@@ -53,11 +59,16 @@ while [[ $# -gt 0 ]]; do
 done
 
 SCRIPT_DIR="$(dirname "$0")"
+DATE=""
```

Comment:
You're erasing DATE after you wrote to it. Was this tested?

Reply:
Sorry, i tested it through configure.py and not directly, fixed now